password/ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for `password`
 
+## 3.0.3.0
+
+-  Added `bcrypt` `defaultParams` used by `hashPassword`
```

Comment:
If you can add credits to this change as well, I think that'll be all for this PR.

```
-  Added `bcrypt` `defaultParams` used by `hashPassword`
  Thanks to [@blackheaven](https://github.com/blackheaven)
    [#69](https://github.com/cdepillabout/password/pull/70)
```